modules/service_container_doctrine/modules/service_container_doctrine_block/service_container_doctrine_block.module

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * Implements hook_block_info().
+ */
+function service_container_doctrine_block_block_info() {
+  $blocks = array();
+  $plugin_manager = \Drupal::service(basename(__FILE__, '.module'). '.block');
+
+  foreach($plugin_manager->getDefinitions() as $definition) {
+    $blocks[$definition['id']]['info'] = $definition['admin_label'];
+  }
+
+  return $blocks;
+}
+
+/**
+ * Implements hook_block_view().
+ */
+function service_container_doctrine_block_block_view($delta = '') {
+  $definition = \Drupal::service(basename(__FILE__, '.module'). '.block')->getDefinition($delta);
```

Comment:
One delta will not be enough for all possible plugins, we need to create real instances of block plugins, like in Drupal 8.

Likely using core blocks with a custom input filter or configuration, have to see what is best performant.